File: app/src/main/java/com/example/kursovaya/MainActivity.java
package com.example.kursovaya;

import android.content.Intent;
import android.os.Bundle;
import android.view.Window;

import androidx.appcompat.app.AppCompatActivity;

import static java.lang.Thread.sleep;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.activity_main);
        Thread logoTimer = new Thread() {
            public void run() {
                try {
                    int logoTimer = 0;
                    while(logoTimer < 1000) {
                        sleep(100);
                        logoTimer = logoTimer +100;
                    };
                    Intent i=new Intent(MainActivity.this,Menu.class);
                    startActivity(i);
                }
                catch (InterruptedException e) {
                    e.printStackTrace();
                }
                finally{
                    finish();
                }
            }
        };
        logoTimer.start();
    }
}

File: app/src/main/java/com/example/kursovaya/MainActivity.kt
package com.example.kursovaya

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent
import android.view.Window

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.activity_main)
        val logoTimer: Thread = object : Thread() {
            override fun run() {
                try {
                    var logoTimer = 0
                    while (logoTimer < 1000) {
                        sleep(100)
                        logoTimer = logoTimer + 100
                    }
                    val i = Intent(this@MainActivity, Menu::class.java)
                    startActivity(i)
                } catch (e: InterruptedException) {
                    e.printStackTrace()
                } finally {
                    finish()
                }
            }
        }
        logoTimer.start()
    }
}